src/Microsoft.Identity.Client/ApiEvent.cs

```diff
@@ -72,6 +72,7 @@ public string UserId
         public bool WasSuccessful
         {
             set => this["was_successful"] = value.ToString();
+            get => this["was_successful"] == "True";
```

Comment:
Hmm, I haven't checked, but I assume the platform API to get the "what is the string literal when a bool true is translated into string", would simply be `true.ToString()`. So yes, I am going to change this line to `... == true.ToString()`.